bigtable/client/client_options.cc

```diff
@@ -28,5 +28,55 @@ ClientOptions::ClientOptions() {
   channel_arguments_ = grpc::ChannelArguments();
 }
 
+void SetCompressionAlgorithm(grpc::ChannelArguments *channel_arguments,
+                             const grpc_compression_algorithm algorithm) {
+  channel_arguments->SetCompressionAlgorithm(algorithm);
+}
+
+void SetGrpclbFallbackTimeout(grpc::ChannelArguments *channel_arguments,
```

Comment:
Will these functions do more in the future? Or are they complete?

I'm curious what their high-level purpose is; it seems that they're basically forwarding calls as-is:

```
void Foo(A* a, B* b) { a->Foo(b); }
```

Why would someone use these functions, instead of just calling `a->Foo(b)` in their code instead of `Foo(a, b)` if they do the same thing?

Reply:
First, per the comment above, the functions should become:

```C++
void Foo(B x) { a_->Foo(x); }
```

Second, I expect some of the functions to do other things, for example, we may not allow users to replace the user-agent string, or may always suffix the library version to the user agent string, or may issue a warning when they replace it (and the code is compiled in debug mode).